app/services/support_interface/application_choices_export.rb

```diff
@@ -68,5 +69,27 @@ def relevant_applications
         .where.not(submitted_at: nil)
         .order('submitted_at asc')
     end
+
+    def format_structured_rejection_reasons(structured_rejection_reasons)
+      return nil if structured_rejection_reasons.blank?
+
+      structured_rejection_reasons
+      .select_high_level_rejection_reasons
+      .keys
+      .map { |reason| format_reason(reason) }
+      .join("\n")
+    end
+
+    def select_high_level_rejection_reasons(structured_rejection_reasons)
+      structured_rejection_reasons.select { |reason, value| value == 'Yes' && reason.include?('_y_n') }
+    end
+
+    def format_reason(reason)
+      reason
+      .split('_')
+      .reject { |string| %w[y n].include?(string) }
+      .join(' ')
+      .humanize
+    end
```

Comment:
Is it clear what's going on in these methods? There's quite a bit of munging going on. Happy to extract some logic out if needed.